tests/integration/sugar/test_transaction.py

```diff
@@ -236,6 +239,35 @@ async def test_payment_high_fee_authorized(self, client):
         self.assertTrue(response.is_successful())
         WALLET.sequence += 1
 
+    @test_async_and_sync(
+        globals(),
+        [
+            "xrpl.transaction.safe_sign_and_autofill_transaction",
+            "xrpl.transaction.submit_transaction",
+        ],
+    )
+    async def test_payment_high_fee_authorized_with_submit_alias(self, client):
+        signed_and_autofilled = await safe_sign_and_autofill_transaction(
+            Payment(
+                account=WALLET.classic_address,
+                sequence=WALLET.sequence,
+                amount="1",
+                # WITH the fee higher than 2 XRP
```

Comment:
I think you can remove these comments in this function. I don't think they add much. 

(Or otherwise the GIVEN comment needs to be when the Payment value is defined)